docs/development/virtualizing_tails.rst

```diff
@@ -0,0 +1,244 @@
+  -                               'pretty_name':  pretty_name
+  -                               }
+  -                    self.status(message)
+  -                    continue
+                   # Skip too small devices, but inform the user
+                   if not info['is_device_big_enough']:
+                       message =_('The device "%(pretty_name)s"'
+
+2. Now run the following two commands:
+
+.. code:: sh
+
+  sudo patch -p0 -d/ < installer.patch
+  sudo /usr/bin/python -tt /usr/lib/tails_installer/tails-installer -u -n --clone -P -m -x
+
+3. The **Tails Installer** will appear. Click **Install Tails**.
+4. Once complete, navigate to **Applications**, **Utilities** and open **Disks**.
+5. Click on the disk named "Tails" and click the Play icon to mount the disk.
+6. Next open ``/media/amnesia/Tails/syslinux/live*.cfg`` and delete all instances
+   of ``live-media=removable``.
+7. Shut down the VM.
+
+Boot to Tails Hard Drive Install
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we will remove the CD and boot to the Tails we just installed on our
+virtual hard drive:
+
+1. Click the VM in the sidebar of VirtualBox and click **Settings**.
+2. Click **Storage** and select the Tails .iso under **Controller: IDE**.
+3. Click the CD icon, then **Remove Disk from Virtual Drive**.
+4. Click **Ok**.
+5. Start the VM.
+
+Configure Persistence
+~~~~~~~~~~~~~~~~~~~~~
+
+1. Configure an admin password when prompted.
+2. Copy the following patch to the Tails VM and save it as ``persistence.patch``:
+
+.. code:: python
+
+   --- /usr/share/perl5/Tails/Persistence/Setup.pm	2017-06-30 09:56:25.000000000 +0000
+   +++ /usr/share/perl5/Tails/Persistence/Setup.pm.mod	2017-07-20 07:17:48.472000000 +0000
+   @@ -404,19 +404,6 @@
+
+        my @checks = (
+            {
+   -            method  => 'drive_is_connected_via_a_supported_interface',
+   -            message => $self->encoding->decode(gettext(
+   -                "Tails is running from non-USB / non-SDIO device %s.")),
+   -            needs_drive_arg => 1,
+   -        },
+   -        {
+   -            method  => 'drive_is_optical',
+   -            message => $self->encoding->decode(gettext(
+   -                "Device %s is optical.")),
+   -            must_be_false    => 1,
+   -            needs_drive_arg => 1,
+   -        },
+   -        {
+                method  => 'started_from_device_installed_with_tails_installer',
+                message => $self->encoding->decode(gettext(
+                    "Device %s was not created using Tails Installer.")),
+
+3. To apply the patch, from the Terminal run:
+
+.. code:: sh
+
+  sudo patch -p0 -d/ < persistence.patch
+
+4. Navigate to **Applications** then **Tails** and click **Configure
+   persistent volume**. Configure a persistent volume enabling all persistence
+   options.
+
+Shared Folders
+~~~~~~~~~~~~~~
+
+1. In the host OS, click on the Tails VM in VirtualBox and then go to
```

Comment:
Nit: "In the host OS" is clearer than "In the host," but still could be improved.